backend/src/api-tests/reference/create.test.ts

```diff
@@ -29,8 +29,22 @@ describe('Creating new reference works', () => {
 
     const { status: getReqStat } = await send<ReferenceDetailsType>(`reference/${createdId}`, 'GET')
     expect(getReqStat).toEqual(200)
+  })
+
+  it('Creation fails without permissions', async () => {
+    logout()
+    const { body: resultBody1, status: getReqStatus1 } = await send('reference/', 'PUT', {
```

Comment:
Small thing, but could these resultBody1 be named by the permissions it has? For example `resultBodyNoPerm` and `resultBodyEr`.